barretenberg/cpp/src/barretenberg/stdlib/primitives/biggroup/biggroup_nafs.hpp

```diff
@@ -485,6 +485,10 @@ std::vector<bool_t<C>> element<C, Fq, Fr, G>::compute_naf(const Fr& scalar, cons
     C* ctx = scalar.context;
     uint512_t scalar_multiplier_512 = uint512_t(uint256_t(scalar.get_value()) % Fr::modulus);
     uint256_t scalar_multiplier = scalar_multiplier_512.lo;
+    // NAF can't handl 0
```

Comment:
Typo here.

Reply:
Curious what is the NAF form of 0. We're doing this in-circuit so we don't benefit in the usual way from using NAF form so it's not so important, though.

Reply:
Merged. Fixing typo here: https://github.com/AztecProtocol/aztec-packages/pull/6965/commits/97fc4505972afbc3706768389fba5e3dc316cd31#diff-b7b31d4897a9c8b424db1f44dcc84a5dbc6f4cb427cc53dab58ca13868d4c13bR488